Refresh.GameServer/Endpoints/Game/Levels/FilterSettings/LevelFilterSettings.cs

```diff
@@ -0,0 +1,120 @@
+using Bunkum.Core;
+using Refresh.GameServer.Extensions;
+using Refresh.GameServer.Services;
+
+namespace Refresh.GameServer.Endpoints.Game.Levels.FilterSettings;
+
+public class LevelFilterSettings
+{
+    /// <summary>
+    /// The game(s) to filter the results to
+    /// </summary>
+    public GameFilterType GameFilterType = GameFilterType.Both;
+    /// <summary>
+    /// Whether or not to display Lbp1 levels in the results (corresponds to gameFilter[]=lbp1)
+    /// </summary>
+    public bool DisplayLbp1 = true;
+    /// <summary>
+    /// Whether or not to display Lbp2 levels in the results (corresponds to gameFilter[]=lbp2)
+    /// </summary>
+    public bool DisplayLbp2 = true;
+    /// <summary>
+    /// Whether or not to display Lbp3 levels in the results (corresponds to gameFilter[]=lbp3)
+    /// </summary>
+    public bool DisplayLbp3 = true;
+    /// <summary>
+    /// Whether or not the user's own levels should be excluded from the results
+    /// </summary>
+    public bool ExcludeMyLevels = false;
+    /// <summary>
+    /// The amount of players to filter for
+    /// eg. only show levels which are compatible with that many players
+    /// </summary>
+    public byte Players = 0;
+    /// <summary>
+    /// The filter setting for move levels
+    /// </summary>
+    public MoveFilterType MoveFilterType = MoveFilterType.True;
+    /// <summary>
+    /// The first label filter
+    /// </summary>
+    public string? LabelFilter0 = null;
+    /// <summary>
+    /// The second label filter
+    /// </summary>
+    public string? LabelFilter1 = null;
+    /// <summary>
+    /// The third label filter
+    /// </summary>
+    public string? LabelFilter2 = null;
```

Comment:
I feel like this should be an array. Any reason this has to be 3 separate fields?

Reply:
no reason to separate it, i'll move it to an array